gravitee-node-vertx/src/main/java/io/gravitee/node/vertx/VertxFactory.java

```diff
@@ -77,54 +88,71 @@ public Vertx getObject() throws Exception {
         LOGGER.debug("Creating a new instance of Vert.x");
         VertxOptions options = getVertxOptions();
 
+        VertxBuilder vertxBuilder = Vertx.builder().with(options);
+
         boolean metricsEnabled = environment.getProperty("services.metrics.enabled", Boolean.class, false);
         if (metricsEnabled) {
             configureMetrics(options);
-        }
 
-        Vertx instance = Vertx.vertx(options);
-        instance.registerVerticleFactory(springVerticleFactory);
+            MeterRegistry promRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
```

Comment:
Out of curiosity, what is the difference/possible impacts compared to using `BackendRegistries.getDefaultNow();`?